### PR TITLE
One html file!

### PR DESCRIPTION
**Pull request by chrisnojima**
Originally I was going to explore generating our html files using webpack html but then decided we could actually have a single html file and better integration w/ electron. 
Instead of packing params in the query string and having to have a way to decode that stuff we instead use the ipc channel to send a bootstrap message over. This allows us to send multiple scripts easily and other parameters.

### CODE REVIEW

File: desktop/app/main-window.js [hunk 1]
```diff
@@ -29,6 +29,19 @@ export default function () {
     }
   )
 
+  const webContents = mainWindow.window.webContents
+  webContents.once('did-finish-load', () => {
+    webContents.send('load', {
```
Comment (chrisnojima) [inline desktop/app/main-window.js]: we pass parameters through the ipc channel and not through the url anymore

File: desktop/renderer/index.js [hunk 2]
```diff
@@ -80,31 +89,52 @@ function setupApp (store) {
...
-  store.dispatch(updateDebugConfig(require('../shared/local-debug-live')))
-})
+function load () {
+  const store = setupStore()
+  setupApp(store)
+  setupHMR(store)
+  render(store, Nav)
+}
+
+window.load = load
```
Comment (chrisnojima) [inline desktop/renderer/index.js]: all bundle js files have a 'load' entry point

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready for review

**Comment by MarcoPolo** [comment]
🚀 

Awesome!!

**Pull request merged**